sdk/management/README.md

```diff
@@ -132,12 +135,78 @@ linuxVM.update()
     .apply();
 ```
 
+### Dependency across Azure resources.
+
+You can create a function app, together with required storage account and app service plan created on specification.
+
+```java
+Creatable<StorageAccount> creatableStorageAccount = azure.storageAccounts()
+    .define(storageAccountName)
+    .withRegion(Region.US_EAST)
+    .withExistingResourceGroup(rgName)
+    .withGeneralPurposeAccountKindV2()
+    .withSku(StorageAccountSkuType.STANDARD_LRS);
+
+Creatable<AppServicePlan> creatableAppServicePlan = azure.appServicePlans()
+    .define(appServicePlanName)
+    .withRegion(Region.US_EAST)
+    .withExistingResourceGroup(rgName)
+    .withPricingTier(PricingTier.STANDARD_S1)
+    .withOperatingSystem(OperatingSystem.LINUX);
+
+FunctionApp linuxFunctionApp = azure.functionApps().define(functionAppName)
+    .withRegion(Region.US_EAST)
+    .withExistingResourceGroup(rgName)
+    .withNewLinuxAppServicePlan(creatableAppServicePlan)
+    .withBuiltInImage(FunctionRuntimeStack.JAVA_8)
+    .withNewStorageAccount(creatableStorageAccount)
+    .withHttpsOnly(true)
+    .withAppSetting("WEBSITE_RUN_FROM_PACKAGE", functionAppPackageUrl)
+    .create();
+```
+
+### Batch Azure resource provisioning
+
+You can batch create and delete managed disk instances.
+
+```java
+List<String> diskNames = Arrays.asList("datadisk1", "datadisk2");
+
+List<Creatable<Disk>> creatableDisks = diskNames.stream()
+    .map(diskName -> azure.disks()
+        .define(diskName)
+        .withRegion(Region.US_EAST)
+        .withExistingResourceGroup(rgName)
+        .withData()
+        .withSizeInGB(1)
+        .withSku(DiskSkuTypes.STANDARD_LRS))
+    .collect(Collectors.toList());
+
+Collection<Disk> disks = azure.disks().create(creatableDisks).values();
+
+azure.disks().deleteByIds(disks.stream().map(Disk::id).collect(Collectors.toList()));
+```
+
+### Integration with Azure role-based access control
+
+You can assign Contributor for an Azure resource to a service principal.
+
+```java
+String raName = UUID.randomUUID().toString();
+RoleAssignment roleAssignment = azure.accessManagement().roleAssignments()
+    .define(raName)
+    .forServicePrincipal(servicePrincipal)
+    .withBuiltInRole(BuiltInRole.CONTRIBUTOR)
+    .withScope(resource.id())
+    .create();
+```
+
```

Comment:
Add a section on RBAC. Probably this is the only lib in java than can do this...